backend/backend.go

```diff
@@ -760,66 +749,27 @@ func (backend *Backend) initPersistedAccounts() {
 // manually user-added). Currently the first bip44 account of BTC/LTC/ETH. ERC20 tokens are added if
 // they were configured to be active by the user in the past, when they could still configure them
 // globally in the settings.
-func (backend *Backend) persistDefaultAccountConfigs(keystore keystore.Keystore) {
+func (backend *Backend) persistDefaultAccountConfigs(keystore keystore.Keystore) error {
 	if backend.arguments.Testing() {
 		if backend.arguments.Regtest() {
-			RBTC, _ := backend.Coin(coinpkg.CodeRBTC)
-			backend.persistBTCAccountConfig(keystore, RBTC,
-				"rbtc",
-				[]scriptTypeWithKeypath{
-					newScriptTypeWithKeypath(signing.ScriptTypeP2WPKHP2SH, "m/49'/1'/0'"),
-					newScriptTypeWithKeypath(signing.ScriptTypeP2PKH, "m/44'/1'/0'"),
-				},
-			)
+			if err := backend.createAndPersistAccountConfig(coinpkg.CodeRBTC, 0, keystore); err != nil {
```

Comment:
if you just `backend.createAndPersistAccountConfig(...)` it'll be simpler:

```go
// Testnets.
if backend.arguments.Testing() {
  if Regtest() {
    return backend.createAndPersistAccountConfig(...)
  }
  for _, coinCode := range []coinpkg.Code{coinpkg.CodeTBTC, coinpkg.CodeTLTC, coinpkg.CodeTETH, coinpkg.CodeRETH} {
    if err := backend.createAndPersistAccountConfig(coinCode, 0, keystore); err != nil {
        return err
    }
  }
  return nil
}

// Mainnets
for _, coinCode := range []coinpkg.Code{coinpkg.CodeBTC,
}
```

Basically less nesting > easier to read: least indented code is the "main" or "production".